cla_frontend/settings/base.py

```diff
@@ -122,6 +122,7 @@ def get_env_value(var_name):
     "wss:",
     "www.google-analytics.com",
     "stats.g.doubleclick.net",
+    "'unsafe-inline'",
```

Comment:
Ah yes Michael I should have mentioned this before. So regarding this line, I did a bit of research and found it's a pretty high security risk in terms of it allows any script to run.

The article I read said to use the hash sequence that gets printed out with the CSP error (it gives us a random hash in the console I believe). That way, only our script tag will get executed as it'll match what's in the CSP

Reply:
Good work, yes, we should absolutely do that, then

Reply:
Done